Analysis/tfcrig/analysis.py

```diff
@@ -639,22 +626,40 @@ def __init__(
         *,
         data_root: str,
         verbose: bool = False,
+        cohorts: list[str] = [],
         mice_of_interest: list[str] = [],
     ) -> None:
         self.data_root = data_root
         self.verbose = verbose
+        self.cohorts = cohorts
         self.mice_of_interest = mice_of_interest
 
         # Keep track of per-file errors
         self.file_errors = {}
 
+        # Use a subset of directories for the analysis
```

Comment:
Filter analysis data based on cohort data, and avoid the `test_data` and `duplicate_data` folders